Content.Server/Abilities/Firestarter/FirestarterComponent.cs

```diff
@@ -0,0 +1,49 @@
+using Robust.Shared.Prototypes;
+using Content.Shared.Damage;
+using Robust.Shared.Audio;
+using Robust.Shared.Serialization.TypeSerializers.Implementations.Custom.Prototype;
+
+namespace Content.Server.Abilities.Firestarter
```

Comment:
file scope namespace

Reply:
did i do well?

Reply:
its still block scoped

Reply:
should it be changed to 

namespace Content.Server.Abilities;
?

Reply:
yes then remove {} and unindent everything

Reply:
should be fixed now :)

Reply:
Did you push the change? It's still block-scoped.